easybuild/easyconfigs/t/tsne/tsne-0.1.8_fix_setup_py.patch

```diff
@@ -0,0 +1,55 @@
+Removed hardcoded flags
```

Comment:
also mention linking to Intel MKL (and don't forget to update the patch checksum if you tweak the patch description)